scripts/functions.py

```diff
@@ -1,4 +1,18 @@
 
+def read_config_bash(paths_all):
+
+    # paths_all must be a list like [a, b, c]
+    conf = {}
+    for path_config in paths_all:
+        # read config
+        with open(path_config) as myfile:
+            for line in myfile:
+                if not line.lstrip().startswith('#'):
+                    name, var = line.partition("=")[::2]
+                    conf[name.strip()] = var.strip()
+    # strip " off values in dictionary conf
+    conf = {k: v.strip('"') for (k, v) in conf.items()}
+    return conf
```

Comment:
I would add a `try` / `catch` to catch issues in situations where the file does not exist